apps/fluent-tester/src/TestComponents/Badge/BasicBadgeTest.tsx

```diff
@@ -12,20 +12,26 @@ import {
   BadgeSize,
   BadgeSizes,
   BadgeIconPosition,
+  BadgeProps,
+  BadgeTokens,
 } from '@fluentui-react-native/badge';
 import { SvgIconProps } from '@fluentui-react-native/icon';
 import { StyledPicker } from '../Common/StyledPicker';
 import { satyaPhotoUrl } from './../PersonaCoin/styles';
 import TestSvg from '../../FluentTester/test-data/test.svg';
 import { ToggleButton } from '@fluentui/react-native';
-import { useFluentTheme, memoize } from '@fluentui-react-native/framework';
+import { useFluentTheme } from '@fluentui-react-native/framework';
 
 const badgeColors: BadgeColor[] = [...BadgeColors];
 const badgeShapes: BadgeShape[] = [...BadgeShapes];
 const badgeSizes: BadgeSize[] = [...BadgeSizes];
 const badgeAppearances: BadgeAppearance[] = [...BadgeAppearances];
 const badgeIconPositions = ['before', 'after'];
 
+const CustomizedBadge = (tokensAndprops: BadgeProps & BadgeTokens) => {
```

Comment:
Our linter might tell you to use "useCustomizedBadge" to make it clear this is a hook.